translations/en.i18n.json

```diff
@@ -56,7 +56,9 @@
       "auto_scroll": "Resume AutoScroll",
       "image_alt": "Slide Preview",
       "list_type": "List",
-      "search_description": "You can search the video captions for specific words or phrases."
+      "search_description": "You can search the video captions for specific words or phrases.",
+      "read_more": "Read more",
```

Comment:
is there an i18n ticket for it?

Reply:
opened ticked for it
https://kaltura.atlassian.net/browse/FEC-13534